src/main/java/frc/robot/commands/BallChucker9000Index.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.BallChucker9000;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+public class BallChucker9000Index extends CommandBase {
```

Comment:
Should extend InstantCommand instead of CommandBase. See Drive-Base-Command-Impl branch for example.